src/main/scala/MacroUtils.scala

```diff
@@ -0,0 +1,75 @@
+package org.cvogt.flow
+
+import scala.language.experimental.macros
+import scala.reflect.macros.blackbox
+
+import org.cvogt.flow.transforms._
+
+trait FlowMacroBase {
+  val c: blackbox.Context
+
+  val allTransforms = Seq[Transform](
+    ImplicitExtractions,
+    PostfixExtractions,
+    ValidateLazy,
+    Normalize,
+    RewriteExtractions
+  )
+}
+
+trait FlowMacroDebug extends FlowMacroBase { base: FlowMacroBase =>
```

Comment:
do we need the cake pattern here or can we just use objects and imports instead? Just easier to understand